libvast/CMakeLists.txt

```diff
@@ -243,6 +243,7 @@ set(tests
   test/save_load.cpp
   test/schema.cpp
   test/serialization.cpp
+  test/span.cpp
```

Comment:
No test for `byte`?

Reply:
Added now. (I left it out initially for the same reason I mentioned above.)